### PR TITLE
Remove {{Obsolete_header}} from Kitchensink

### PR DESCRIPTION
**Pull request by bershanskiy**
This commit removes the last instances of {{Obsolete_header}} from EN-US content.

### CODE REVIEW

File: files/en-us/mdn/kitchensink/index.html [hunk 1]
```diff
@@ -365,7 +363,7 @@ <h2 id="Types">Types</h2>
 </dl>
 
 {{Non-standard_Header}}
-{{Obsolete_Header}}
+{{SeeCompatTable}}
```
Comment (peterbe) [inline files/en-us/mdn/kitchensink/index.html:366]: Why did you add this one?

Comment (peterbe) [reply files/en-us/mdn/kitchensink/index.html:366]: Actually, I think this is a mistake. That macro is already used on line 294.

**Review by peterbe — CHANGES_REQUESTED** [review]
I think an unrelated new macro slipped in. Otherwise; great!

**Review by Elchi3 — APPROVED** [review]
I wanted to ask Peter if it is fine to change kitchensink, but if Peter is happy, I'm happy.

**Comment by peterbe** [comment]
> I wanted to ask Peter if it is fine to change kitchensink, but if Peter is happy, I'm happy.

The rule simple; you can't make the Kitchensink have flaws. 

We have a piece of CI in Yari that builds the Kitchensink and asserts that it's flawless. It's a technique to augment the Kumascript unit tests with some end-to-end tests. 
Because of the harsh reality of kumascript being weird and old we decided that a busted kumascript should NOT block the build. I.e. if an error happens in the execution of a kuma script we don't crash the build but instead "log" it as a `MacroExecutionError` flaw. 
So if you accidentally introduce a bug in an `.ejs` file, that the kumascript unit tests don't catch, it's still proceed to build the Kitchensink page but with flaws. 

So basically, you can edit it as much as you want as long as it's exercising as many (relevant and somewhat not obsolete) macros as possible. ...without triggering flaws.

**Comment by bershanskiy** [comment]
Thank you for prompt response!
I removed the unnecessary `{{SeeCompatTable}}`.

**Review by peterbe — APPROVED** [review]
